src/main/java/be/minelabs/item/items/GasPotion.java

```diff
@@ -56,6 +46,12 @@ public ItemStack getDefaultStack() {
     @Override
     public TypedActionResult<ItemStack> use(World world, PlayerEntity user, Hand hand) {
```

Comment:
I'm not sure whether we even support this, but dispensers have a separate callback somewhere that also needs to take the gamerule into account (again if we already support gas potions in dispensers)

Reply:
At the moment we do not support this.